system/web/flash/ColdboxCacheFlash.cfc

```diff
@@ -59,10 +59,7 @@ component extends="coldbox.system.web.flash.AbstractFlashScope" accessors="true"
 		else if ( isSessionDefined and structKeyExists( session, "URLToken" ) ) {
 			return prefix & session.URLToken;
 		} else {
-			throw(
-				message = "Cannot find a jsessionid, URLToken or cfid/cftoken in the cookie scope. Please verify",
-				type    = "ColdboxCacheFlash.CFIDException"
-			);
+			return false;
```

Comment:
Ok, I understand the point.  I think however, relying on boolean constructs might be problematic an create extra checks that lead to performance.  After thinking of the issue, where we are stateless, I think it's best to create a unique identifier we can track for the entire request.  Because with this approach flash becomes useless.  You still might want the flash in order to last the request itself. Something like this

```
else if ( isNull( request.cbFlashId ) ){
   request.cbFlashId = createUUID();
} else {
   return request.cbFlashId;
}
```

At least this way, we still produce a key that will last for the request.

Reply:
Updated